apps/www/src/app/challenge-accepted/components/teasers/article-teaser.tsx

```diff
@@ -76,7 +74,7 @@ export const ArticleTeaser = async ({ path, image }: ArticleProps) => {
   // and extract the pathname from it.
   const url = new URL(path, process.env.NEXT_PUBLIC_BASE_URL)
 
-  const { data }: { data: ArticleQueryResult } = await getClient().query({
+  const { data } = await getClient().query<ArticleTeaserQuery>({
```

Comment:
In the `src/graphql/republik-api/article.query` file we should be using the generated `gql` function now instead of the one from `@apollo/client`, so this query ist typed automatically, no?

Reply:
We need to do the same thing in `newsletter-teaser.tsx`